packages/effect/src/stream/Sink.ts

```diff
@@ -454,6 +474,14 @@ export const fromEffect = <A, E, R>(
     (a) => Pull.halt<End<A>>([a])
   ))))
 
+/**
+ * Drains elements from the stream by ignoring all inputs.
+ *
+ * @since 2.0.0
+ * @category constructors
+ */
+export const drain: Sink<void, unknown> = fromChannel(Channel.drain(Channel.identity()) as any)
```

Comment:
This doesn't look quite right. You will need to use Channel.mapDone to set an output of `void`.

Reply:
Got it - I was going off the v3 version, but I think our type signatures weren't quite as good there. Thanks for pushing a fix 👍